libraries/core/src/main/kotlin/com/bumble/appyx/core/node/Node.kt

```diff
@@ -230,6 +230,10 @@ open class Node(
     private fun handleUpNavigationByPlugins(): Boolean =
         plugins<UpNavigationHandler>().any { it.handleUpNavigation() }
 
+    protected fun ensureNodeWasBuilt() {
```

Comment:
We already have this check inside lifecycle listener of the node, it does not work?

Reply:
nope, we are not using NodeHost for view testing which propagates lifecycle events. Lifecycle never makes it to `onCreate` to enable that check